harness/testTypedArray.js

```diff
@@ -20,16 +20,17 @@ var floatArrayConstructors = [
   Float32Array
 ];
 
-var intArrayConstructors = [
+var nonClampedIntArrayConstructors = [
```

Comment:
I guess that `nonClampedIntArrayConstructors` should be mentioned in the `defines` list above.

Reply:
Good catch, done.